include/flatbuffers/reflection_generated.h

```diff
@@ -77,6 +81,8 @@ inline const BaseType (&EnumValuesBaseType())[18] {
     Union,
     Array
   };
+  static_assert(sizeof(values) / sizeof(BaseType) == CountBaseTypes,
```

Comment:
How will the static_asserts() be added if this is a generated file? You would have to modify the generator logic to emit these statements as well.

Reply:
Right. This got pointed by Wouter as well. I'm addressing that in another change.

Reply:
To be clear, I won't add static checks now that I realized that these arrays are generated - I expect we can rely on the generation logic to create proper arrays here.